pkg/Cpanel/Security/Advisor/Assessors/Trojans.pm

```diff
@@ -73,7 +91,618 @@ sub _check_for_libkeyutils {
     );
 
     return 1;
+}
+
+sub _check_for_UMBREON_rootkit {
+    my ($self) = @_;
+    my $dir = '/usr/local/__UMBREON__';
+    my $dir2 = '/usr/local/UMBREON';
+    if ( chdir $dir or chdir $dir2 ) {
```

Comment:
I recommend `if ( -d $dir or -d $dir2 )`, as this won't modify the working directory for the whole assessor.